src/page-components/CartItem/CartItem.tsx

```diff
@@ -0,0 +1,86 @@
+import { BasketItemWithProduct } from "@/services/ecommerce/generated/v0";
+import { styled } from "@mitodl/smoot-design";
+import { Card } from "../../components/Card/Card";
+import { createTheme } from "@mitodl/smoot-design";
+import { Typography } from "@mui/material";
+
+import StyledCard from "../../components/Card/StyledCard";
+
+const theme = createTheme();
+
+type CartItemProps = {
+  item: BasketItemWithProduct;
+};
+
+const CartItemContainer = styled.div(() => ({
+  display: "flex",
+  marginTop: "16px",
+  marginBottom: "8px",
+}));
+
+const CartItemImage = styled.img`
+  width: 192px;
+  margin: 0;
+`;
+
+const CartItemContentContainer = styled.div`
+  width: auto;
+  flex-grow: 1;
+  margin: 0;
+  margin-left: 32px;
+`;
+
+const CartItemProductMeta = styled.div`
+  display: flex;
+  align-items: start;
+  margin-bottom: 8px;
+`;
+
+const CartItemProductPrice = styled.div`
+  margin-left: auto;
+  ${{ ...theme.typography.h5 }},
+`;
+
+const CartItemProductSku = styled.div`
+  ${{ ...theme.typography.body1 }},
+  color: ${theme.custom.colors.silverGrayLight},
```

Comment:
**Request:** commas should be semi-colons here and above. Will see if a linter can pick it up.